10_datastructures_lab/01.py

```diff
@@ -1,10 +1,20 @@
-"""
-Write a program that takes two strings
-from the user and checks if they represent
-a valid user name.
-Valid users and passwords:
-    apple => red
-    lettuce => green
-    lemon => yellow
-    orange => orange
-"""
+_author_ = 'mariag'
+
+import sys
+
+file_hosts = 'hosts.txt'
+hostsDic = {}
+
+#load key|value to dictionary
+with open(file_hosts, 'r') as f_hosts:
+    for line in f_hosts:
+        host = line.split('=')[0] 
+        ip = line.rstrip('\n').split('=')[1]
+        hostsDic[host] = ip
+
+for host_input in sys.argv[1:]:
+    value = hostsDic.get(host_input,0)
+    if value == 0:
+        print "host:'%s' doesn't exist" % host_input
+    else:
+        print value

```

Comment:
nice
This is a more readable version of it:
```
        host, ip = line.rstrip('\n').split('=')
        hostsDic[host] = ip
```
Also it is clearer and more Pythonic to write

```if host_input in hostDic:``` instead of lines 16-17

Lastly, please use .format instead of % in line 18.